Modules/_sqlite/microprotocols.c

```diff
@@ -38,17 +38,17 @@ static PyObject *psyco_adapters = NULL;
 int
 pysqlite_microprotocols_init(PyObject *module)
 {
+    int res;
+
     /* create adapters dictionary and put it in module namespace */
     if ((psyco_adapters = PyDict_New()) == NULL) {
         return -1;
     }
 
-    if (PyModule_AddObject(module, "adapters", psyco_adapters) < 0) {
-        Py_DECREF(psyco_adapters);
-        return -1;
-    }
+    res = PyModule_AddObjectRef(module, "adapters", psyco_adapters);
```

Comment:
Declare res here to make the code shorter.

```suggestion
    int res = PyModule_AddObjectRef(module, "adapters", psyco_adapters);
```

Reply:
I agree, but it violates PEP 7, no? But there's a lot of PEP 7 violations in the _sqlite3 module anyways, so why not :)

Reply:
Python now uses C99, it's a great pleasure to write C99 compared to ISO C89!

Reply:
Perfect!

Reply:
Fixed in 590d11d

Reply:
Please update also pysqlite_microprotocols_init().

Reply:
You didn't address this remark.

Reply:
Did it in 911ee5a ;)